Makefile

```diff
@@ -85,6 +86,10 @@ image-release:
 	aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin $(IMAGE_TAG_OWNER) 
 	docker buildx build --platform linux/arm64,linux/amd64 --output=type=registry --tag $(IMAGE) -f ./build/Dockerfile .
 
+init-image-release:
```

Comment:
Do we really need new image? We could use the same percona operator image but move to https://hub.docker.com/layers/library/debian/bullseye-20231009-slim/images/sha256-f6514f25a34c18b7450ca0a26f5669e7810823ba4d08a0c177e863b965591676?context=explore rather than using the distroless